apps/demos/Demos/Splitter/Overview/ReactJs/PaneContent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const dimensionOptions = new Set(['size', 'minSize', 'maxSize']);
+const getPaneState = (data) => {
+  if (data.resizable !== false && !data.collapsible) {
+    return 'Resizable only';
+  }
+  const resizableText = data.resizable ? 'Resizable' : 'Non-resizable';
+  const collapsibleText = data.collapsible ? 'collapsible' : 'non-collapsible';
+  return `${resizableText} and ${collapsibleText}`;
+};
+const getFilteredDimensionOptions = (data) =>
```

Comment:
Let's add figure bracket to make this function more explicit?
Let's also add empty lines to separate different functions

Reply:
ESLint doesn't allow the use of curly braces when returning a value without other statements.

```Let's also add empty lines to separate different functions```
This is autogenerated content. Check the React version of this demo. The line breaks are presented there.